packages/core/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -47,7 +47,7 @@ export class DropdownMenu {
   }
 
   connectedCallback() {
-    this.button.setAttribute("aria-haspopup", "true");
+    this.button.setAttribute("aria-haspopup", "menu");
```

Comment:
Dit moet nog wel verwerkt worden in het CSS component `packages\css\src\components\dropdown-menu\dropdown-menu.template.ts`, en daarvoor moet ook volgens mij het type van de `ariaHaspopup` eigenschap nog wijzigen, want nu wordt die als volgt doorgezet:
`aria-haspopup=${ifDefined(ariaHaspopup?.toString())}`
... terwijl dat dus helemaal geen boolean is. Maar, zoals op https://www.digitala11y.com/aria-haspopup-properties/ te lezen, een van de waarden 
token
false (default)
true
menu
listbox
tree
grid

Reply:
@benheldoorn Heb jij dit opgelost in jouw laatste commit, of is dat iets anders wat erop lijkt? Ik zie namelijk wel een union type met al deze waarden van Ad

Reply:
Ja, als het goed is wordt nergens meer `ariaHasPopup` als boolean gebruikt

Reply:
Super!